spec/poker/deck_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+
+RSpec.describe Poker::Deck, type: :model do
+  describe '#initialize' do
+    it 'has class variable of cards' do
+      deck = Poker::Deck.new
+
+      expect(deck.cards).to_not be_nil
+      expect(deck.cards).to be_kind_of(Array)
+    end
+    it 'receives create on initialize' do
+      expect_any_instance_of(Poker::Deck).to receive(:create!)
```

Comment:
Stubbing methods on the class under test isn't generally good. Why is this important?